docs/configuration/index.md

```diff
@@ -1186,6 +1191,15 @@ The following properties pertain to segment metadata caching on the Overlord tha
 |`druid.manager.segments.useIncrementalCache`|Denotes the usage mode of the segment metadata incremental cache. Possible modes are: (a) `never`: Cache is disabled. (b) `always`: Reads are always done from the cache. Service start-up will be blocked until cache has synced with the metadata store at least once. Transactions will block until cache has synced with the metadata store at least once after becoming leader. (c) `ifSynced`: Reads are done from the cache only if it has already synced with the metadata store. This mode does not block service start-up or transactions.|`never`|
 |`druid.manager.segments.pollDuration`|Duration (in ISO 8601 format) between successive syncs of the cache with the metadata store. This property is used only when `druid.manager.segments.useIncrementalCache` is set to `always` or `ifSynced`.|`PT1M` (1 minute)|
 
+##### Auto-kill unused segments (Experimental)
+
+These configs pertain to the new embedded mode of running [kill tasks on the Overlord](../data-management/delete.md#auto-kill-data-on-the-overlord-experimental).
+
```

Comment:
I think it would be good to explicitly call out that none of the coordinator-based auto-kill configs apply to this overlord-based auto-kill feature (even in the combined coordinator+OL mode). Bringing this up because I had that assumption initially when reviewing https://github.com/apache/druid/pull/18028. The reasons are already cross-linked in `data-management/delete.md`, so it should be clear for anyone curious.

Reply:
updated